test/Java9andUp/src_latest/org/openj9/test/attachAPI/VmIdGetter.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+package org.openj9.test.java9AttachAPI;
+/* This class is separated from the main test because we need two
+ * copies to accommodate old and new versions of ProcessHandle
+ */
+public class VmIdGetter {
+
+	static String getVmId() {
+		long myPid = ProcessHandle.current().pid();
+		String myId = Long.toString(myPid);
+		return myId;
+	}
+
+}
```

Comment:
We should put this file in test/Java9andUp/src to avoid duplication in both ssrc_latset and src_current.

Reply:
Sadly no.  the ProcessHandle API changed from getPid() to pid();